browser/net/brave_network_delegate_browsertest.cc

```diff
@@ -188,7 +212,7 @@ IN_PROC_BROWSER_TEST_F(BraveNetworkDelegateBrowserTest, DefaultCookiesBlocked) {
 IN_PROC_BROWSER_TEST_F(BraveNetworkDelegateBrowserTest,
                        PrefToggleBlockAllToBlockThirdParty) {
   DefaultBlockAllCookies();
-  BlockThirdPartyCookies();
+  BlockAllThirdPartyCookies();
```

Comment:
Renamed so that it's consistent with other helper functions like `DefaultBlockAllCookies`